demo/app.jsx

```diff
@@ -1,8 +1,8 @@
 /*global document:false window:false*/
-import React from "react";
+import React, { Component, PropTypes } from "react";
```

Comment:
oh! I like `Component` so much better than `Chart` for the catch all set of utils